ijson/backends/_yajl2.c

```diff
@@ -0,0 +1,942 @@
+/*
+ * C extension to bind yajl to ijson
+ *
+ * Contributed by Rodrigo Tobar <rtobar@icrar.org>
+ *
+ * ICRAR - International Centre for Radio Astronomy Research
+ * (c) UWA - The University of Western Australia, 2016
+ * Copyright by UWA (in the framework of the ICRAR)
+ * All rights reserved
```

Comment:
Mmmm, good point, I haven't given much thought to that particular phrase really, I simply added it from the template I've seen in other codes of ours without realizing it conflicted with the license. I will have to bring this up and come back with an answer.